test/perf/PerfHelper.hpp

```diff
@@ -27,14 +27,14 @@
 // Define performance main, with default opt level of 3, and scan PERF_ARGS to
 // override default onnx-mlir compiler options.
 #define PERF_MAIN()                                                            \
-  const std::string envArgName("PERF_ARGS");                                   \
-  const std::string O3("3");                                                   \
-                                                                               \
   int main(int argc, char **argv) {                                            \
     ::benchmark::Initialize(&argc, argv);                                      \
-    onnx_mlir::omSetCompilerOption(                                            \
-        onnx_mlir::OptionKind::CompilerOptLevel, O3.c_str());                  \
-    if (onnx_mlir::omSetCompilerOptionsFromEnv(envArgName.c_str()) != 0)       \
+    int onnxMlirArgc = 2;                                                      \
```

Comment:
This causes build breaks when compiling on Windows. I noticed we don't build these tests on Windows right now, so I'd like to add them to the CI build to avoid another quiet failure like that, but before that we'll need a fix for the failure as well.

The specific issue is with `onnxMlirArgc`:

```
D:\sources\onnx-mlir-mainline\test\perf\PerfGemm.cpp(124): error C2131: expression did not evaluate to a constant
D:\sources\onnx-mlir-mainline\test\perf\PerfGemm.cpp(124): note: failure was caused by a read of a variable outside its lifetime
D:\sources\onnx-mlir-mainline\test\perf\PerfGemm.cpp(124): note: see usage of 'onnxMlirArgc'
D:\sources\onnx-mlir-mainline\test\perf\PerfGemm.cpp(124): error C3863: array type 'const char *[onnxMlirArgc]' is not assignable
```

but I wonder why `PERF_MAIN` is a macro and not a function call? Same with the other functions in this file.